setup.py

```diff
@@ -58,6 +58,7 @@ def get_requirements():
     packages=find_packages(exclude=('tests',)),
     data_files=[
         ('/etc/resallocserver', ['config/pools.yaml', 'config/server.yaml']),
+        ('/var/www', ['resallocwebui/cgi-resalloc']),
```

Comment:
This is probably causing that %py2_install takes this file, too - and breaks the epel7 build.